HLTrigger/Configuration/python/customizePhase2HLTTracking.py

```diff
@@ -0,0 +1,728 @@
+    minHitsPerNtuplet = cms.uint32(3),
+    onGPU = cms.bool(True),
+    phiCuts = cms.vint32(
+        716, 841, 736, 467, 999,
+        423, 909, 862, 983, 950,
+        735, 479, 542, 420, 974,
+        987, 465, 423, 629, 822,
+        449, 462, 851, 922, 848,
+        422, 743, 963, 966, 561,
+        572, 415, 628, 438, 945,
+        437, 934, 743, 653, 412,
+        773, 790, 952, 932, 459,
+        516, 472, 928, 655, 479,
+        971, 983, 788, 982, 497
+    ),
+    pixelRecHitSrc = cms.InputTag("siPixelRecHitsCUDA"),
+    ptCut = cms.double(0.8500000238418579),
+    ptmin = cms.double(0.8999999761581421),
+    trackQualityCuts = cms.PSet(
+        maxChi2 = cms.double(5),
+        maxTip = cms.double(0.3),
+        maxZip = cms.double(12),
+        minPt = cms.double(0.5)
+    ),
+    useRiemannFit = cms.bool(False),
+    useSimpleTripletCleaner = cms.bool(True),
+    z0Cut = cms.double(12.851884460665062)
+
+)
+
+    from RecoTracker.PixelTrackFitting.pixelTrackSoAFromCUDAPhase2_cfi import pixelTrackSoAFromCUDAPhase2 as _pixelTracksSoAPhase2
+    process.pixelTracksSoA = SwitchProducerCUDA(
+        # build pixel ntuplets and pixel tracks in SoA format on the CPU
+        cpu = _pixelTracksCUDAPhase2.clone(
+            pixelRecHitSrc = "siPixelRecHitsCPU",
+            idealConditions = False,
+            onGPU = False,
+            includeJumpingForwardDoublets = True,
+            dupPassThrough = False,
+	        minHitsPerNtuplet = 4,
+            z0Cut = 9.5,
+            hardCurvCut = 0.012,
+            ptCut = 1,
+        ),
+        cuda = _pixelTracksSoAPhase2.clone()
+    )
+
+    from RecoTracker.PixelTrackFitting.pixelTrackProducerFromSoAPhase2_cfi import pixelTrackProducerFromSoAPhase2 as _pixelTrackProducerFromSoAPhase2
+    process.pixelTracks = _pixelTrackProducerFromSoAPhase2.clone(
+        pixelRecHitLegacySrc = "siPixelRecHits",
+        beamSpot = "hltOnlineBeamSpot"
+    )
+
+    process.pixelTracksTask = cms.Task(
+        process.pixelTracksCUDA,
+        process.pixelTracksSoA,
+        process.pixelTracks
+    )
+
+    process.HLTTrackingV61Task = cms.Task(process.MeasurementTrackerEvent, 
+                                          process.generalTracks, 
+                                          process.highPtTripletStepClusters, 
+                                          process.highPtTripletStepHitDoublets, 
+                                          process.highPtTripletStepHitTriplets, 
+                                          process.highPtTripletStepSeedLayers, 
+                                          process.highPtTripletStepSeeds, 
+                                          process.highPtTripletStepTrackCandidates, 
+                                          process.highPtTripletStepTrackCutClassifier, 
+                                          process.highPtTripletStepTrackSelectionHighPurity, 
+                                          process.highPtTripletStepTrackingRegions, 
```

Comment:
For some reason, this line throws an error about "highPtTripletStepTrackingRegions not included in the process". This seems weird, given that the module exists:
https://github.com/cms-hlt-upgrade-trk/cmssw/blob/phase2_hlt_Tracking_133_dev/HLTrigger/Configuration/python/HLT_75e33/modules/highPtTripletStepTrackingRegions_cfi.py
Maybe it is not properly included in an "upstream" configuration?

In any case, adding the lines:
```
from HLTrigger.Configuration.HLT_75e33.modules.highPtTripletStepTrackingRegions_cfi import highPtTripletStepTrackingRegions as _highPtTripletStepTrackingRegions
process.highPtTripletStepTrackingRegions = _highPtTripletStepTrackingRegions.clone()
```
in line 409 solves the issue.

Reply:
For releases >= 13_1_1 some renaming(cleaning of hlt tracking modules has been done:
the modules pixelTracksTrackingRegions and highPtTripletStepTrackingRegions
are now renamed hltPhase2PixelTracksAndHighPtStepTrackingRegions

you can simply include hltPhase2PixelTracksAndHighPtStepTrackingRegions in your task rather than cloning it
(otherwise the work to avoid duplicate modules is vanished)

similarly the following changes should be applied (here and in phase2TrackingValidation_cff):
hltPixelTracks --> hltPhase2PixelTracks (not mandatory but would avoid duplicate)
hltPixelVertices --> hltPhase2PixelVertices ([commented below](https://github.com/cms-hlt-upgrade-trk/cmssw/pull/1#pullrequestreview-1663070075))

Reply:
reference for the changes: https://github.com/cms-sw/cmssw/pull/42491

Reply:
Thanks for the explanation and pointers. Since the situation is as you describe, I guess we arrive at the same conclusion, i.e. that this PR cannot work out of the box in `CMSSW_13_3_0_pre3`, and that an update is needed so that it works as per the PR description.